docs/source/handlers.rst

```diff
@@ -22,6 +22,12 @@ string into a data structure in ``response_data`` that is used when
 evaluating ``response_json_paths`` entries in a test or doing
 JSONPath-based ``$RESPONSE[]`` substitutions.
 
+A YAML content handler has been added to extend the JSON handler.
```

Comment:
We need to call it something different than the YAML content handler because what the name `YAMLContentHandler` means is that this is a handler for request and/or response bodies which are YAML. That's not what this is.

Naming is hard, so I don't have any clever ideas at the moment but will think on it.

Reply:
I went with the ugly but explicit route :/ YAMLDiskLoadingJSONHandler . Alternatives could be YDLJSONHandler or YDLJHandler for short? Any other suggestions are welcome too.